src/VisualStudio/Core/Def/Implementation/ProjectSystem/FileChangeWatcher.cs

```diff
@@ -136,7 +145,10 @@ public Context(FileChangeWatcher fileChangeWatcher, string directoryFilePath)
                     _directoryFilePathOpt = directoryFilePath;
 
                     _fileChangeWatcher.EnqueueWork(
-                        service => { ErrorHandler.ThrowOnFailure(service.AdviseDirChange(_directoryFilePathOpt, fWatchSubDir: 1, this, out _directoryWatchCookie)); });
+                        async service =>
+                        {
+                            _directoryWatchCookie = await service.AdviseDirChangeAsync(_directoryFilePathOpt, watchSubdirectories: true, this).ConfigureAwait(false);
```

Comment:
i thought you said above we weren't listening for directories.  can you clarify the split?

Reply:
FileChangeWatcher still supports it -- I just removed the call to the directory watch at the site for now.